src/lambda/activation.ts

```diff
@@ -37,7 +38,16 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
                     functionNode: node,
                     outputChannel,
                 })
-        )
+        ),
+        // Capture debug finished events, and delete the base build dir if it exists
+        vscode.debug.onDidTerminateDebugSession(async session => {
```

Comment:
Is `baseBuildDir` common between all executions? If so, does this fail or break something if you have more than one SAM debug running at a time?

Reply:
it's a per execution directory (it makes a new temp directory every time) so it's safe to delete one while running another